src/types/state.js

```diff
@@ -37,10 +37,18 @@ export type Reducer<T> = (T | void, Action) => T;
 export type SymbolicationStatus = 'DONE' | 'SYMBOLICATING';
 export type ThreadViewOptions = {|
   +selectedCallNodePath: CallNodePath,
-  +rightClickedCallNodePath: CallNodePath | null,
   +expandedCallNodePaths: PathSet,
   +selectedMarker: MarkerIndex | null,
-  +rightClickedMarker: MarkerIndex | null,
+|};
+
+export type RightClickedCallNodePath = {|
```

Comment:
I think the type shouldn't be "...CallNodePath" because it's not _just_ a call node path. A call node path is something very specific in our code, this is an array of function indexes, that uniquely identifies a specific call node. So this is confusing that we see `RightClickedCallNodePath` but it's really an object containing things.
I'd suggest `RightClickedCallNode`.
The same remark goes to the reducer/state property name as well.